semconv/experimental/src/main/scala/org/typelevel/otel4s/semconv/experimental/DbExperimentalAttributes.scala

```diff
@@ -59,18 +59,35 @@ object DbExperimentalAttributes {
     "db.cassandra.speculative_execution_count"
   )
 
-  /** The name of the primary Cassandra table that the operation is acting upon,
-    * including the keyspace name (if applicable).
+  /** Deprecated, use `db.collection.name` instead.
+    */
+  @deprecated("Use `db.collection.name` instead", "0.5.0")
```

Comment:
sorry I'm so late with pointing this out (was sick), but the `since` version is wrong

Reply:
The value is hardcoded in the template and we cannot determine it dynamically :( 
https://github.com/typelevel/otel4s/blob/main/buildscripts/semantic-convention/templates/SemanticAttributes.scala.j2#L88

I guess the best option is to use an empty string instead. At least we will not mislead users.